.github/workflows/docker-release.yml

```diff
@@ -0,0 +1,36 @@
+name: Docker Release
+
+on:
+  workflow_dispatch:
+
+# Ensure single workflow runs at a time
+# Runs that are already underway will be canceled when a new job is queued
+# See <https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#concurrency>
+concurrency:
+  group: release-${{ github.ref }}
+  cancel-in-progress: false
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: qemu setup
```

Comment:
Is `qemu` required here? If so, for what?

Reply:
Possibly not, but I have not ran a pipeline to do multiarch builds without it.